tests/client-sdk/safety/run_tests.md

```diff
@@ -0,0 +1,33 @@
+# Using Llama Stack as Library with Ollama
+=
+```
+LLAMA_STACK_CONFIG=tests/client-sdk/safety/resources/ollama.yaml pytest tests/client-sdk/safety -v
+```
+
+# Using Llama Stack as Library with Together
```

Comment:
Maybe we should have a single doc for testing instructions?

Reply:
Yeah that's a good idea, I have another PR and it should be 2 more coming. I will consolidate those as I go through them. Does that sound good?

Reply:
Thanks! SG!